### PR TITLE
Add rules when add database with auto apply is called

### PR DESCRIPTION
**Pull request by KavishShah09**

### CODE REVIEW

File: space-cli/cmd/modules/addons/database.go [hunk 1]
```diff
@@ -206,6 +206,54 @@ func keepSettingConfig(token, dbType string, account *model.Account, v *model.Sp
...
+				},
+				Spec: map[string]interface{}{
+					"rule": "allow",
+				},
+			}
+			if err := operations.ApplySpec(token, account, v); err != nil {
+				logrus.Warningln("Couldn't add eventing rules", nil)
+				continue
+			}
+
```
Comment (sharadregoti) [inline space-cli/cmd/modules/addons/database.go:256]: You have to set prepared queries rule not eventing

File: space-cli/cmd/modules/addons/database.go [hunk 2]
```diff
@@ -206,6 +206,54 @@ func keepSettingConfig(token, dbType string, account *model.Account, v *model.Sp
...
+			v = &model.SpecObject{
+				API:  "/v1/config/projects/{project}/database/{dbAlias}/collections/{col}/rules",
+				Type: "db-rules",
+				Meta: map[string]string{
+					"dbAlias": v.Meta["dbAlias"],
+					"col":     "default",
+					"project": v.Meta["project"],
+				},
+				Spec: map[string]interface{}{
+					"isRealtimeEnabled": v.Spec.(map[string]interface{})["enabled"],
```
Comment (sharadregoti) [inline space-cli/cmd/modules/addons/database.go:219]: Set it to false

File: space-cli/cmd/modules/addons/database.go [hunk 3]
```diff
@@ -206,6 +206,54 @@ func keepSettingConfig(token, dbType string, account *model.Account, v *model.Sp
...
+							"rule": "allow",
+						},
+						"update": map[string]interface{}{
+							"rule": "allow",
+						},
+					},
+				},
+			}
+			if err := operations.ApplySpec(token, account, v); err != nil {
+				logrus.Warningln("Couldn't add default rules", nil)
```
Comment (sharadregoti) [inline space-cli/cmd/modules/addons/database.go:237]:
```suggestion
				logrus.Warningln("Couldn't add default collection rules", nil)
```

File: space-cli/cmd/modules/addons/database.go [hunk 4]
```diff
@@ -206,6 +206,54 @@ func keepSettingConfig(token, dbType string, account *model.Account, v *model.Sp
...
+				Meta: map[string]string{
+					"project": v.Meta["project"],
+					"id":      "default",
+				},
+				Spec: map[string]interface{}{
+					"rule": "allow",
+				},
+			}
+			if err := operations.ApplySpec(token, account, v); err != nil {
+				logrus.Warningln("Couldn't add eventing rules", nil)
```
Comment (sharadregoti) [inline space-cli/cmd/modules/addons/database.go:253]:
```suggestion
				logrus.Warningln("Couldn't add default prepared query rules", nil)
```

**Comment by KavishShah09** [comment]
I had run go mod tidy before pushing @sharadregoti

**Comment by sharadregoti** [comment]
Fixes #1066

**Pull request merged**